lib/engine/index.ts

```diff
@@ -337,6 +367,8 @@ export default class AssistantEngine extends Tp.BaseEngine {
      */
     run() : Promise<void> {
         this._running = true;
+        if (this._activityMonitor)
+            this._activityMonitor.start();
```

Comment:
You can add _activityMonitory to the list of engine modules and start/stop will be called when the engine initializes (in open) and deinitializes (in close).